### DISCUSSION TITLE
贵州的太黑暗了，直接把微信群给封杀了！！！！！！！！

### DISCUSSION THREAD

**Discussion by ewkyrtas** [WeNeedHome/SummaryOfLoanSuspension#1073] [General]
[贵州的太黑暗了，直接把微信群给封杀了！！！！！！！！](https://github.com/WeNeedHome/SummaryOfLoanSuspension/pull/1068)

**Comment by ghost** [comment]
地方政府根本没有能力解决这个烂摊子了，现在就等着把楼炸了，还要还贷款，花了钱买了个空，笑死🤣

**Reply by ewkyrtas** [reply]
他们一天坐起不好好  监管 结果让我们来承担
